packages/backend/src/services/UserService.ts

```diff
@@ -333,18 +333,16 @@ export class UserService extends BaseService {
         const existingUserWithEmail = await this.userModel.findUserByEmail(
             email,
         );
-        if (
-            existingUserWithEmail &&
-            existingUserWithEmail.organizationUuid !== organizationUuid
-        ) {
-            throw new ParameterError(
-                'Email is already used by a user in another organization',
-            );
-        }
-        if (existingUserWithEmail && existingUserWithEmail.isActive) {
-            throw new ParameterError(
-                'Email is already used by a user in your organization',
-            );
+        if (existingUserWithEmail && existingUserWithEmail.organizationUuid) {
```

Comment:
so this now means that existing users that haven't yet joined an org (i.e. organizationUuid is `undefined`) will fall through here to account setup

Reply:
correct